packages/suite/src/utils/suite/oauth.ts

```diff
@@ -11,11 +12,8 @@ export const getOauthReceiverUrl = () => {
     if (!window.ipcRenderer) {
         return `${window.location.origin}${getPrefixedURL('/static/oauth/oauth_receiver.html')}`;
     }
-
-    // todo: upload oauth_receiver.html from suite to trezor.io (it contains code to handle ipcRenderer);
-    // return 'https://beta-wallet.trezor.io/oauth_receiver.html';
-    // todo: temporarily herokuapp is used.
-    return 'https://track-suite.herokuapp.com/oauth/';
+    // for desktop. but this solution is temporary, local http server will be used later to accept callback
+    return 'https://wallet.trezor.io/oauth_receiver.html';
```

Comment:
I would suggest prefixing temporary fix comments with a searchable term like `TEMP:` to make it easier to find and report.